src/components/Onboarding/steps.ts

```diff
@@ -0,0 +1,261 @@
+        step: {
+          ...commonStepOptions,
+          target: '#word-by-word-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('inline-wbw'),
+        step: {
+          ...commonStepOptions,
+          target: '#word-by-word-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('translations'),
+        step: {
+          ...commonStepOptions,
+          target: '#translation-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+        },
+      },
+    ],
+    [OnboardingGroup.READING_EXPERIENCE]: [
+      {
+        ...getDetails('surah-info'),
+        step: {
+          ...commonStepOptions,
+          target: '#surah-info-button',
+          placement: 'left-start',
+        },
+      },
+      {
+        ...getDetails('listen'),
+        step: {
+          ...commonStepOptions,
+          // first element with class play-verse-button
+          target: '.play-verse-button:first-child',
+          spotlightClicks: true,
+          showNextButton: false,
+          showPrevButton: false,
+        },
+      },
+      {
+        ...getDetails('select-reciter'),
+        step: {
+          ...commonStepOptions,
+          target: '#audio-player-overflow-menu-trigger',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('ayah-tafsir'),
+        step: {
+          ...commonStepOptions,
+          target: '.tafsir-verse-button',
+        },
+      },
+      {
+        ...getDetails('ayah-reflection'),
+        step: {
+          ...commonStepOptions,
+          target: '.reflection-verse-button',
+        },
+      },
+      {
+        ...getDetails('3-dot-menu'),
+        step: {
+          ...commonStepOptions,
+          target: '.overflow-verse-actions-menu-trigger',
+        },
+      },
+      {
+        ...getDetails('reading-view'),
+        step: {
+          ...commonStepOptions,
+          target: '#reading-preference-switcher',
+          spotlightClicks: true,
+        },
+      },
+    ],
+    [OnboardingGroup.PERSONALIZED_FEATURES]: [
+      {
+        ...getDetails('login'),
+        step: {
+          ...commonStepOptions,
+          target: '#login-button',
+        },
+      },
+    ],
+  };
+};
+
+export const onboardingChecklist = (t: Translate) => [
+  {
+    group: OnboardingGroup.HOMEPAGE,
+    title: t('onboarding:checklist.1'),
+    href: '/',
+  },
+  {
+    group: OnboardingGroup.SETTINGS,
+    title: t('onboarding:checklist.2'),
+    href: '/',
+  },
+  {
+    group: OnboardingGroup.READING_EXPERIENCE,
+    title: t('onboarding:checklist.3'),
+    href: getSurahNavigationUrl(1),
+  },
+  {
+    group: OnboardingGroup.PERSONALIZED_FEATURES,
+    title: t('onboarding:checklist.4'),
+    href: '/login',
```

Comment:
This should be conditionally done. It still redirects me to login page even though I am logged in.